portal-impl/src/com/liferay/portlet/documentlibrary/service.xml

```diff
@@ -385,6 +385,7 @@
 		<column name="defaultFileEntryTypeId" type="long" />
 		<column name="hidden" type="boolean" />
 		<column name="restrictionType" type="int" />
+		<column name="allowedPropagation" type="boolean" />
```

Comment:
I'm quite concerned about the term `allowedPropagation` here. Let me try to explain the idea:
- We're proposing a new term in this feature. It should tell other developers what it is about. Maybe, i think we're not English native speaking so we just try to put a term that is understandable to us internally. But eventually we need to let others understand this new term, especially when this term will be proposed to BChan ;-) i believe we should try to make it more self-explained.
- Reading through the ACs, we sound to add something to store the idea of "permission propagation" (if the folder is a parent folder) and also "permission inheritance" (if the folder is a child). More or less, we may think of a term that can represent in some way both ideas of "propagation" and "inheritance". I'm still not knowing of any English word that can do the job though, maybe we need to ask for some advice from another native English speaker. The only candidate comes to my mind could be something like "permissionPropagation" (or "permissionPropagationEnabled" if boolean value needs to be an adjective).

Reply:
Hi @hhtank, 
Thanks for your comments, I will make it more self-explained.
I think it is better to change to the "permissionPropagationEnabled"  fields.
Thanks,
Loc